app/components/preprint-title-editor.js

```diff
@@ -33,10 +33,10 @@ const TitleValidation = buildValidations({
  * @class preprint-title-editor
  */
 
-export default Ember.Component.extend(TitleValidation, {
+export default Component.extend(TitleValidation, {
     title: null,
     titlePlaceholder: 'Enter preprint title',
-    isValid: Ember.observer('title', function() {
+    isValid: observer('title', function() {
```

Comment:
The observer here could be removed as well. Again this may belong to another ticket.